internal/helm/helm.go

```diff
@@ -438,3 +441,31 @@ func resolveChartVersion(versionFlag string) (semver2.Version, *chart.Chart, err
 	}
 	return version, helmChart, nil
 }
+
+// UpgradeCurrentRelease upgrades the existing Helm release with given Helm values.
+func UpgradeCurrentRelease(
+	ctx context.Context,
+	namespace string,
+	helmValues map[string]interface{},
+	resetValues bool,
```

Comment:
nit: Having too boolean arguments next to each other is likely to cause issues down the line, perhaps consider using a struct for some of these arguments, especially anything that's not required (namespace, helmValues, resetValues, reuseValues)

Reply:
yeah makes sense, i'll add a struct for helm-related parameters 👍